assets/locales/ru/translation.json

```diff
@@ -5,115 +5,117 @@
+  "Check for updates": "Проверить обновления",
   "New Session Window…": "Новое окно сессии…",
-  "Configurations…": "Конфигурация…",
+  "Configurations…": "Конфигурации…",
   "Hide Appium": "Скрыть Appium",
-  "Hide Others": "Скрыть другие",
+  "Hide Others": "Скрыть все, кроме Appium",
   "Show All": "Показать все",
   "Quit": "Выход",
-  "Toggle Full Screen": "Переключение полноэкранного режима",
+  "Toggle Full Screen": "Вкл/выкл полноэкранный режим",
   "Languages": "Языки",
   "View": "Вид",
   "Reload": "Перезагрузка",
-  "Toggle Developer Tools": "Переключение инструментов разработчика",
-  "Learn More": "Подробнее",
+  "Toggle Developer Tools": "Вкл/выкл инструменты разработчика",
+  "Learn More": "Узнать больше",
   "Undo": "Отмена",
   "Cloud Providers": "Облачные провайдеры",
   "Environment Variables": "Переменные среды",
   "Save and Restart": "Сохранить и перезапустить",
-  "Application must be restarted for changes to take effect": "Приложение должно быть перезапущено для вступления изменений в силу",
+  "Application must be restarted for changes to take effect": "Для обновления необходим перезапуск",
   "Restart Now": "Перезапустить",
   "Restart Later": "Перезапустить позже",
   "Simple": "Простой",
-  "Advanced": "Доп. настройки",
+  "Advanced": "Экспертный",
   "Presets": "Пресеты",
-  "Edit Configurations": "Редактирование конфигураций",
+  "Edit Configurations": "Изменить конфигурации",
   "Server Address": "Адрес сервера",
   "Server Port": "Порт сервера",
   "Logfile Path": "Путь к файлу журнала",
-  "Log Level": "Уровень журнала",
-  "Override Temp Path": "Переопределить путь временных файлов",
+  "Log Level": "Детализация журнала",
+  "Override Temp Path": "Переопределить путь ко временным файлам",
   "Node Config File Path": "Путь к файлу конфигурации Node",
   "Local Timezone": "Местный часовой пояс",
   "Allow Session Override": "Разрешить переопределение сессии",
   "Log Timestamps": "Записывать метки времени в журнал",
-  "Supress Log Color": "Supress Log Color",
-  "Strict Caps Mode": "Strict Caps Mode",
-  "Relaxed Security": "Relaxed Security",
+  "Supress Log Color": "Текст журнала одним цветом",
+  "Strict Caps Mode": "Строгий режим для возможностей устройств",
+  "Relaxed Security": "Без дополнительной защиты",
   "Default Capabilities": "Возможности по умолчанию",
   "WebDriverAgent Port": "Порт WebDriverAgent",
-  "executeAsync Callback Host": "executeAsync Callback Host",
-  "executeAsync Callback Port": "executeAsync Callback Port",
+  "executeAsync Callback Host": "executeAsync Callback Хост",
+  "executeAsync Callback Port": "executeAsync Callback Порт",
   "Bootstrap Port": "Порт Bootstrap",
   "Selendroid Port": "Порт Selendroid",
   "Chromedriver Port": "Порт Chromedriver",
-  "Chromedriver Binary Path": "Путь Chromedriver Binary",
-  "Save Server Arguments Preset": "Save Server Arguments Preset",
+  "Chromedriver Binary Path": "Расположение chromedriver.exe",
```

Comment:
`.exe` is strict to the Windows platform, so it is definitely not the best update.